packages/apps/dev-wallet/src/App/layout-mini.tsx

```diff
@@ -25,7 +31,12 @@ export const LayoutMini: FC = () => {
           </Link>
         </Text>
       </Stack>
-      <main className={mainStyle}>
+      <main
+        className={mainStyle}
+        style={{
+          backgroundImage: `radial-gradient(circle farthest-side at 50% 170%, ${accentColor}, transparent 75%)`,
```

Comment:
this could be done this way or with VE variants, we can keep it this way for now